Frontend/src/cse-ui-kit/CreateContentBlock_button/CreateContentBlock.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { StyledButton, buttonProps } from './CreateContentBlock-Styled';
+import { AiFillEdit } from "react-icons/ai";
+
+type Props = {
+  children?: React.ReactElement | any;
+  onClick?: (...args: any) => void;
+} & buttonProps;
+
+export default function CreateContentBlock({ children, onClick, ...styleProps }: Props) {
+  return (
+    <StyledButton
+      onClick={onClick}
+      {...styleProps}
+    >
+      {children}
+    </StyledButton>
+  );
```

Comment:
the button should not take in children, it should just import the AiFillEdit icon in this layer, check out how laura does the buttons and copy the same format